portfolio/src/main/webapp/index.html

```diff
@@ -172,14 +172,14 @@ <h2 class="comments-header">Comments</h2>
         <option value="all">All comments</option>
         <option value="none">No comments</option>
       </select>
-      <button 
-        id="comment-delete-button" 
-        onClick="deleteComments()" 
+      <button
+        id="comment-delete-button"
+        onClick="deleteComments()"
         class="comments-delete"
       >
         Delete All Comments
       </button>
-      <div id="comments-section"></div>
+      <div id="comments-section" class="comments-section"></div>
```

Comment:
Use same string for id and class is a little weird to me. Can you rename one of them? Or maybe just use ID selector in css?

Reply:
Done